limitador/src/storage/redis/redis_cached.rs

```diff
@@ -44,6 +44,7 @@ pub struct CachedRedisStorage {
     async_redis_storage: AsyncRedisStorage,
     redis_conn_manager: ConnectionManager,
     partitioned: Arc<AtomicBool>,
+    flush_tx: tokio::sync::watch::Sender<()>,
```

Comment:
Wondering if a `Notify` wouldn't be more appropriate, correct me if I'm wrong, but this will enqueue every single change and have the loop on "the other side" redo all the work... while it'd race against updates in the shared DS...

Reply:
Yeah it does look simpler.  Watch channels don't enqueue, they only store the last value.

Reply:
Ah, right, they do. And `Notify` sorts of behave similarly. So that now... with the timeout being `0` this will probably loop like crazy, other than the system being idle. The batch will be of the size of the race. So thinking we should notify "smartly"... wdyt?

Reply:
Updated the PR to use Notify.